.devcontainer/Dockerfile

```diff
@@ -0,0 +1,15 @@
+# See here for image contents: https://github.com/microsoft/vscode-dev-containers/tree/v0.162.0/containers/typescript-node/.devcontainer/base.Dockerfile
+
+# [Choice] Node.js version: 14, 12, 10
+ARG VARIANT="14-buster"
+FROM mcr.microsoft.com/vscode/devcontainers/typescript-node:0-${VARIANT}
+
+RUN apt-get update && export DEBIAN_FRONTEND=noninteractive \
+    && apt-get -y install --no-install-recommends python3-venv
```

Comment:
Had to additionally install `python3-venv` here as I was seeing the following error:
```
codespace ➜ /workspaces/cfn-lint-action (main ✗) $ node index.js 
[command]/usr/bin/python3 --version
Python 3.8.5
[command]/usr/bin/python3 -m venv /tmp/setup-cfn-lint-kTkNbV/.venv
The virtual environment was not created successfully because ensurepip is not
available.  On Debian/Ubuntu systems, you need to install the python3-venv
package using the following command.

    apt-get install python3-venv

You may need to use sudo with that command.  After installing the python3-venv
package, recreate your virtual environment.

Failing command: ['/tmp/setup-cfn-lint-kTkNbV/.venv/bin/python3', '-Im', 'ensurepip', '--upgrade', '--default-pip']

::error::The process '/usr/bin/python3' failed with exit code 1
::error::The process '/usr/bin/python3' failed with exit code 1
```

Reply:
Yup that's fine 👍 It would be needed so good shout putting it in 👍